--- freenx-0.7.1/node.conf.sample.orig	2007-10-15 00:25:14.000000000 +0200
+++ freenx-0.7.1/node.conf.sample	2008-05-20 19:03:12.000000000 +0200
@@ -394,7 +394,13 @@
 
 # The key that contains the name of the complete path of command name
 # 'xauth'.
-#COMMAND_XAUTH=/usr/X11R6/bin/xauth
+COMMAND_XAUTH=xauth
+
+COMMAND_XSET=xset
+
+COMMAND_XMODMAP=xmodmap
+
+COMMAND_XKBCOMP=xkbcomp
 
 # The key that contains the name of the complete path of command name
 # 'smbmount'.
@@ -413,7 +419,7 @@
 #COMMAND_SMBUMOUNT_CIFS=/sbin/umount.cifs
 
 # The key that contains the name of the complete path of the 'netcat' command.
-#COMMAND_NETCAT=netcat
+COMMAND_NETCAT=nc
 
 # The key that contains the name of the complete path of the 'ssh' and
 # 'ssh-keygen' command.
@@ -476,7 +482,7 @@
 # for examples of useful parameters.
 #AGENT_EXTRA_OPTIONS_RFB=""
 #AGENT_EXTRA_OPTIONS_RDP=""
-#AGENT_EXTRA_OPTIONS_X=""
+AGENT_EXTRA_OPTIONS_X="-fp /usr/share/fonts/misc/,/usr/share/fonts/100dpi/,/usr/share/fonts/75dpi/,/usr/share/fonts/Type1/,/usr/share/fonts/TTF/"
 
 # The number of seconds we wait for the nxagent to start before
 # deciding startup has failed
@@ -498,6 +504,8 @@
 # Extra options to nxproxy. See !M documentation for useful parameters.
 #PROXY_EXTRA_OPTIONS=""
 
+APPLICATION_LIBRARY_PRELOAD="/usr/lib/libX11-nx.so.6:/usr/lib/libXext-nx.so.6:/usr/lib/libXcomp.so.2:/usr/lib/libXcompext.so.2:/usr/lib/libXrender-nx.so.1"
+
 # In case you want to use an external 'rdesktop' command
 # set this to "1".
 # 
